src/+datools/+statistical/+ensemble/EnKF.m

```diff
@@ -3,14 +3,21 @@
 
     methods
 
-        function analysis(obj, R, y)
-
+        function analysis(obj, observation)
+            %ANALYSIS   Method to overload the analysis function
+            %
+            %   ANALYSIS(OBJ) assimilates the current observation with the
+            %   background/prior information to get a better estimate
+            %   (analysis/posterior)
+            
             inflation = obj.Inflation;
```

Comment:
I am also thinking that inflation shouldnt be a part of analysis. but something done in the running script. It is a heuristic that works, and should be outside the filter. Same goes for rejuvenation. 

If i was doing an enkf with inflation and rejuvenation, I will have to only have a separate rejuvenation outside of the analysis. If I wanted to inflate ETPF, I will have to inflate outside the analysis.To be consistent, I think we should do all inflations, rejuvenations, etc outside the anlaysis. What do you think @AndreyAPopov and @Abhat-93

Reply:
For the EnKF inflation is NOT a heuristic. I have shown that it is a required part of the filter. For other filters it is a heuristic, and sometimes a bad one.

Likewise rejuvenation is not a heuristic. I understand we have a lot of code duplication for it right now, but there is no good way to do it without later breaking other filters that use both those things differently. I say leave it the way it is.

Reply:
Also, are you saying we should do enkf.inflate, then enkf.analysis(obs) ? No way. It should be part of the analysis in the code.

Reply:
If you think so.